ping/ping_common.c

```diff
@@ -83,6 +83,8 @@ void usage(void)
 		"  -V                 print version and exit\n"
 		"  -w <deadline>      reply wait <deadline> in seconds\n"
 		"  -W <timeout>       time to wait for response\n"
+		"\nRRI options (Huawei):\n"
```

Comment:
What `RRI` means? Why do you mention Huawei?

Reply:
The reason is to have the -A and -3 options in the same binary to see the actual RTT deviation on the interface.